lib/internal/module.js

```diff
@@ -50,6 +51,40 @@ function stripBOM(content) {
   return content;
 }
 
+/**
+ * Find end of shebang line and slice it off
+ */
+function stripShebang(content) {
```

Comment:
As a single data point: I've been a UNIX user for 20+ years and this is the first time I've heard someone call it a hashbang.